satviz-serial/src/test/java/edu/kit/satviz/serial/IntSerialBuilderTest.java

```diff
@@ -0,0 +1,42 @@
+package edu.kit.satviz.serial;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.concurrent.ThreadLocalRandom;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class IntSerialBuilderTest {
+
+  private IntSerializer serial = new IntSerializer();
+
+  @Test
+  void testSomeInts() {
+    int[] ints = new int[]{0, 1, -1, 2, 3, 4, 5, 42, 100, 200, -3333, 1000000000, -1000000000};
+    for (int i : ints) {
+      assertDoesNotThrow(() -> testSingleInt(i));
+    }
+  }
```

Comment:
Das gilt für alle Tests dieser Art: `assertDoesNotThrow` würde ich hier nicht verwenden, da es die assertions in der Hilfsmethode "überschattet" und `IOException`s außerdem keine Test-Fehler, sondern Fehler in der Testumgebung sind (in diesem Fall). 

Stattdessen rate ich zu diesem Pattern:
```suggestion
  void testSomeInts() throws IOException {
    int[] ints = new int[]{0, 1, -1, 2, 3, 4, 5, 42, 100, 200, -3333, 1000000000, -1000000000};
    for (int i : ints) {
      try {
        testSingleInt(i);
      } catch (SerializationException e) {
        fail(e);
      }
    }
  }
```

Reply:
Die Tests werfen nur noch IOException, SerializationException wird mit fail() abgefangen.